src/backend/impress/settings.py

```diff
@@ -367,8 +370,46 @@ class Base(Configuration):
                 environ_name="API_USERS_LIST_THROTTLE_RATE_BURST",
                 environ_prefix=None,
             ),
+            "document": values.Value(
+                default="80/minute",
+                environ_name="API_DOCUMENT_THROTTLE_RATE",
+                environ_prefix=None,
+            ),
+            "document_access": values.Value(
+                default="50/minute",
+                environ_name="API_DOCUMENT_ACCESS_THROTTLE_RATE",
+                environ_prefix=None,
+            ),
+            "template": values.Value(
+                default="30/minute",
+                environ_name="API_TEMPLATE_THROTTLE_RATE",
+                environ_prefix=None,
+            ),
+            "template_access": values.Value(
+                default="30/minute",
+                environ_name="API_TEMPLATE_ACCESS_THROTTLE_RATE",
+                environ_prefix=None,
+            ),
+            "invitation": values.Value(
+                default="60/minute",
+                environ_name="API_INVITATION_THROTTLE_RATE",
+                environ_prefix=None,
+            ),
+            "document_ask_for_access": values.Value(
+                default="30/minute",
+                environ_name="API_DOCUMENT_ASK_FOR_ACCESS_THROTTLE_RATE",
+                environ_prefix=None,
+            ),
+            "config": values.Value(
+                default="30/minute",
+                environ_name="API_CONFIG_THROTTLE_RATE",
+                environ_prefix=None,
+            ),
```

Comment:
Could this be in this list as well ? 
https://github.com/suitenumerique/docs/blob/0ac9f059b6212781000c3d490d1ae7885cdef067/src/backend/impress/settings.py#L662-L671